### ISSUE TITLE
Many bugs with custom properties

### ISSUE THREAD

**Issue by tvld** [elreco/vue-tailwind-datepicker#57]
Type bugs:

```ts
const customShortcuts = () => {
    return [
      {
        label: 'Last Years',
        atClick: (): string => {
          const date = new Date()
          return [new Date(date.setFullYear(date.getFullYear() - 1)), new Date()]
        }
      }
    ] as unknown as undefined // >>>>>>> should not be needed !!
  }
```

**Comment by elreco** [comment]
You adding a string type on "atClick" function return, you must replace by a `Date[]` type


The type error is fixed on the v1.2.10

**Comment by tvld** [comment]
ok, thanks, was confused as the v-model was string.
